orb/core/context.py

```diff
@@ -369,6 +370,20 @@ def scope(self):
         return self.raw_values['scope']
 
     @property
+    def system(self):
```

Comment:
added the ability to act upon a system (default to the global instance).  useful for testing or if you have a complex database environment